lua/example_of_dynamic_config_request.lua

```diff
@@ -1,10 +1,13 @@
 -- TODO Add examples for other operations. For example:
--- Functions to add/del/list the GK FIB entries.
 -- Functions to list the ARP table.
 -- Functions to list the ND table.
 -- Functions to process the GT policies.
 -- ......
 
+require "dylib"
```

Comment:
I think we need to add the following lines just above `require "dylib"`:
```
-- The dylib module is already loaded, but it only contains
-- C functions statically linked. 
-- Unsetting its package.loaded entry allows Lua to load
-- the Lua functions of the module.
package.loaded["dylib"] = nil
```
I derived the lines above from the analogous lines in `lua/gatekeeper_config.lua` when it requires module "gatekeeper".

If this works, I think we should use `module(..., package.seeall)` in dylib as well. I don't remember the why of this line is in `lua/gatekeeper.lua`, but both modules have the same need: having static Lua C functions and Lua code.

Reply:
After adding the lines above, there is a new error message `` ./lua/dylib.lua:63: ';' expected near '=' at line 2!``.

Reply:
This error message suggests that `lua/dylib.lua` wasn't being loaded before. Now that it's being loaded, it seems to have a syntax error.